portfolio/src/main/webapp/getServletResponse.js

```diff
@@ -1,13 +1,26 @@
-/*
- * Fetch a JSON array of messages from the server and randomly choose one to display
+
+/**
+ * gets the JSON array of submitted emails from the server
```

Comment:
Nit: Capital letter. Same everywhere when starting a comment.

Reply:
Yes I made this change